circuit_maintenance_parser/__init__.py

```diff
@@ -81,72 +79,9 @@ def get_provider_class_from_sender(email_sender: str) -> Type[GenericProvider]:
     return provider_parser
 
 
-def get_provider_data_types(provider_name: str) -> Iterable[str]:
-    """Returns the expected data types for each provider."""
-    provider_name = provider_name.lower()
-    for provider in SUPPORTED_PROVIDERS:
-        if provider.get_provider_type() == provider_name:
-            break
-    else:
-        raise NonexistentParserError(
-            f"{provider_name} is not a currently supported provider. Only {', '.join(SUPPORTED_PROVIDER_NAMES)}"
-        )
-
-    return provider.get_data_types()
-
-
-# TODO: In Release 1.2.0, we introduced new Provider abstraction to accomodate multiple potential parser per provider.
-# In order to mantain backwards compatibility until a new major version release (2.x.x) we still mantain the following
```

Comment:
So just to confirm - after merging this PR, the next release will be 2.0.0 since these are breaking changes?

Reply:
yes, this is the idea